java-api/src/main/java/com/sap/cloud/security/token/Token.java

```diff
@@ -45,6 +44,15 @@ static Token create(String jwt) {
 		if (services.isEmpty()) {
 			throw new ProviderNotFoundException("No TokenFactory implementation found in the classpath");
 		}
+		if (services.size() > 2) {
+			throw new ProviderException("More than 2 TokenFactory service providers found. There should be only one");
```

Comment:
"more than one customer TokenFactory" ... the default doe not count,

Reply:
good point, thx 💯